src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```diff
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net5.0;net6.0;net7.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;net7.0;net9.0;netstandard2.0</TargetFrameworks>
```

Comment:
wondering why there was no .Net8 here?

Reply:
We chose to not do it at that time :) Will be fixing in the 2x version.